src/Collector/CollectedData/CollectedResolvedNode.php

```diff
@@ -7,21 +7,24 @@
 use PHPStan\ShouldNotHappenException;
 
 /**
- * @phpstan-type CollectedResolvedNodeArray array{resolver: string, params: array<string, string>}
+ * @phpstan-type CollectedResolvedNodeArray array{resolver: string, analysedFile: string, params: array<string, string>}
```

Comment:
This was added because of first custom resolver so it's not needed for actual state

Reply:
No it is needed with custom resolvwrs too to let analyser know where to collect context. For example if tempalte belongs to some control in vendor you need to pass file with that control. Otherwise it would not be collected for latte context

Reply:
In Node collectors. File where node is located is root of latte context analysis - and we extracted it from array keys used by PHPStan in CollectedDataNode. BUt for custom resolvers we need way to say it explicitly to be able to collect context from files not matched by any node-based collector.